src/main/proto/th2_grpc_common/common.proto

```diff
@@ -93,6 +104,10 @@ message RawMessageBatch {
   repeated RawMessage messages = 1;
 }
 
+message MessageGroupBatch {
```

Comment:
I would name it a bit differently. 
```suggestion
message CombinedMessageBatch {
```

Reply:
I dunno `CombinedMessageBatch` implies to me that it's a batch of combined messages, but it's in fact a batch of combined message groups

Reply:
Probably, you are right. It is a batch of message groups that have combined messages inside. So, let's leave the name as it is now.